brainstormforce/ultimate-addons-for-gutenberg
# Fix: Marketing Button - Gap between icon and title not working on frontend.

### Description

- https://app.startinfinity.com/b/Mi5zu3VRVzk/M7Dpq9yR4Pe/cccf580c-482a-4990-8576-b35bfbbaf352?redirect=true&view=0ce18087-e6b2-4a91-b54c-c386ce1f811c&t=attributes

### Screenshots


### Types of changes




- Bug fix (non-breaking change which fixes an issue)

### How has this been tested?

- Drag and drop marketing buttons.
- Check Gap between Icon and Title setting backend as well as frontend.
- Check by changing icon position before/after.
- Check by adding color and hover color are all reflecting on the front or not.

### Checklist:
